components/sup/src/manager/service/hooks.rs

```diff
@@ -213,6 +213,14 @@ impl Hook for PostRunHook {
         }
     }
 
+    fn retry(&self, exit_value: Self::ExitValue) -> bool {
+        match exit_value.0 {
+            0 => false,
+            1 => true,
+            _ => false,
+        }
```

Comment:
Why not
```rust
        exit_value.0 == 1
```
?

Even better, define a constant (at the `fn` or type level, either seems good) and you get:
```rust
        exit_value == SHOULD_RETRY
```

You'll have to derive `PartialEq` for `ExitCode`, but that seems worth it to me.

Also, unrelated, if feels like this method would more accurately be named `should_retry`.